e2e/testcases/reconciler_manager_test.go

```diff
@@ -92,25 +92,37 @@ func TestManagingReconciler(t *testing.T) {
 			hasReplicas(managedReplicas))
 	})
 
-	// test case 3: override the memory limit has been moved to TestAutopilotReconcilerAdjustment
+	// test case 3: override the resource limits/requests and manually change the resource limits/requests has been moved to TestAutopilotReconcilerAdjustment
```

Comment:
Do we still need to keep this comment here? The test is in a different function.

Reply:
Removed the comment.